### PR TITLE
EPMLABSBRN-1118 Added article into info page

### PR DESCRIPTION
**Pull request by Mikleprog**
EPMLABSBRN-1118 Added article into info page

### CODE REVIEW

**Review by ElenaSpb — APPROVED** [review]

File: frontend/app/templates/description.hbs [hunk 1]
```diff
@@ -108,17 +108,21 @@
...
-          <a
-            href={{t "description.links.articles.link"}}
-            target="_blank"
-            rel="noopener noreferrer"
-            class="hover:no-underline text-blue-light underline"
-          >
-            {{t "description.links.articles.link_title"}}
-          </a>
+          <ul class="mt-3 ml-8 space-y-3 list-disc list-inside">
+            {{#each (range 0 1 true) as |i|}}
```
Comment (lifeart) [inline frontend/app/templates/description.hbs:112]: let's replace `(range 0 1 true)` to `(array 0 1)`

Comment (Mikleprog) [reply frontend/app/templates/description.hbs]: Done

File: frontend/translations/ru-ru.yaml [hunk 2]
```diff
@@ -292,19 +296,19 @@ contributors:
   title: "Наша команда"
   subtitle: "Мы собрались все вместе, чтобы сделать вашу жизнь проще!"
   people:
-    0: 
+    0:
```
Comment (ElenaSpb) [inline frontend/translations/ru-ru.yaml:299]: please avoid such empty changes

**Review by lifeart — APPROVED** [review]

**Comment by lifeart** [comment]
@Mikleprog thank you for contribution!

**Pull request merged**